a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -12,697 +12,189 @@
             new_port = self.neutron_client.create_port(port)
-        except Exception as e:
-            message = _('Error creating neutron port on network '
-                        '{network_id}.').format(
-                network_id=load_balancer.vip.network_id)
+            new_port = utils.convert_port_dict_to_model(new_port)
+        except Exception:
+            message = "Error creating port in network: {0}".format(network_id)
             LOG.exception(message)
-            raise base.AllocateVIPException(
-                message,
-                orig_msg=getattr(e, 'message', None),
-                orig_code=getattr(e, 'status_code', None),
-            )
-        new_port = utils.convert_port_dict_to_model(new_port)
-        return self._port_to_vip(new_port, load_balancer)
+            raise exceptions.PortCreationFailedException(message)
+        return new_port
 
-    def unplug_aap_port(self, vip, amphora, subnet):
-        interface = self._get_plugged_interface(
-            amphora.compute_id, subnet.network_id, amphora.lb_network_ip)
-        if not interface:
-            # Thought about raising PluggedVIPNotFound exception but
-            # then that wouldn't evaluate all amphorae, so just continue
-            LOG.debug('Cannot get amphora %s interface, skipped',
-                      amphora.compute_id)
-            return
+    def delete_port(self, port_id):
         try:
-            self.unplug_network(amphora.compute_id, subnet.network_id)
+            self.neutron_client.delete_port(port_id)
         except Exception:
-            pass
-        try:
-            aap_update = {'port': {
-                'allowed_address_pairs': []
-            }}
-            self.neutron_client.update_port(interface.port_id,
-                                            aap_update)
-        except Exception:
-            message = _('Error unplugging VIP. Could not clear '
-                        'allowed address pairs from port '
-                        '{port_id}.').format(port_id=vip.port_id)
+            message = "Error deleting port: {0}".format(port_id)
             LOG.exception(message)
-            raise base.UnplugVIPException(message)
-
-        # Delete the VRRP port if we created it
-        try:
-            port = self.get_port(amphora.vrrp_port_id)
-            if port.name.startswith('octavia-lb-vrrp-'):
-                self.neutron_client.delete_port(amphora.vrrp_port_id)
-        except (neutron_client_exceptions.NotFound,
-                neutron_client_exceptions.PortNotFoundClient):
-            pass
-        except Exception as e:
-            LOG.error('Failed to delete port.  Resources may still be in '
-                      'use for port: %(port)s due to error: %s(except)s',
-                      {'port': amphora.vrrp_port_id, 'except': e})
 
-    def unplug_vip(self, load_balancer, vip):
+    def create_ve_port(self, ip, subnet):
```

Comment:
This isn't required. Important piece is `fixed_ips` being set in the code below; however, we can do that just by modifying the `fixed_ip` argument to be `fixed_ips` in method signature of `create_port`. It'll accept `{'ip_address': ip, 'subnet_id': subnet.id}` mappings.